### PR TITLE
Update tools and dependencies

### PR DESCRIPTION
**Pull request by kennyklaviyo**
# Purpose

**Type of PR**
* [ ] Bug Fix
* [ ] Feature Work
* [ ] Revert
* [ ] Refactor / Code Cleanup
* [ ] Other (fill in...)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,10 +9,10 @@ buildscript {
 
     repositories {
         google()
-        jcenter()
+        mavenCentral()
```
Comment (ndurell) [inline build.gradle:12]: This one is better now?

Comment (kennyklaviyo) [reply build.gradle:12]: jcenter is no more

File: versions.gradle [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
+    gradleBuildToolsVersion = '7.3.1'
+    gradleKotlinVersion = '1.7.20'
     dokkaVersion = '0.10.1'
 
     // android versioning
-    androidCompileVersion = 29
-    buildToolsVersion = '30.0.2'
+    androidCompileVersion = 33
+    buildToolsVersion = '30.0.3'
     minSDKVersion = 19
```
Comment (ndurell) [inline versions.gradle:10]: https://twitter.com/minsdkversion?lang=en

File: build.gradle (hunk 1 above)
Comment (kennyklaviyo) [reply build.gradle:12]: image

**Review by ndurell — APPROVED** [review]

File: versions.gradle (hunk 2 above)
Comment (kennyklaviyo) [reply versions.gradle:10]: nice

**Pull request merged**